src/addons/attach/attach.js

```diff
@@ -11,12 +11,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = attach(require('../../dist/xterm'));
+    module.exports = attach(require('../../xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../dist/xterm'], attach);
+    define(['../../xterm'], attach);
```

Comment:
I think it's OK to merge this into `typescript_build_2` and move forward with sourcelair#296 after we ensure nothing breaks.

Considering sourcelair#335 I think eventually it's a good time to start migrating to a type safe repository with TypeScript gradually. I still have a few questions, but I will do a more detailed writeup within the day.

I think we can pursue it by **2.2**.